src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Refactoring/RazorCodeActionContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.AspNetCore.Razor.Language;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Refactoring
+{
+    struct RazorCodeActionContext
+    {
+        public readonly CodeActionParams Request;
```

Comment:
To be consistent with the rest of the repo I'd put these below the constructor and turn them into properties:
```C#
public CodeActionParams Request { get; }
```

Even though they're `get` only you can still set them in the ctor.